scipy/linalg/_decomp_qz.py

```diff
@@ -78,7 +78,7 @@ def _qz(A, B, output='real', lwork=None, sort=None, overwrite_a=False,
         raise ValueError("The 'sort' input of qz() has to be None and will be "
                          "removed in a future release. Use ordqz instead.")
 
-    if output not in ['real', 'complex', 'r', 'c']:
+    if output.lower() not in ['real', 'complex', 'r', 'c']:
```

Comment:
Comparing to e.g. https://github.com/scipy/scipy/blob/v0.18.1/scipy/linalg/decomp_schur.py#L90 it seems that that `linalg` doesn't always do this. Why do it here?

Reply:
It's just a habit I guess. I can revert it back I really don't mind. Just picked up from some tutorial a while back which stayed.

Reply:
Not like it's a big deal. But maybe if it is to be changed it would be better to them all at once.